packages/litexa/src/parser/lambda-preamble.coffee

```diff
@@ -8,134 +8,11 @@
-          #console.log "fetched from DB", JSON.stringify(data.Item)
-          wasInitialized = data.Item?.data?
-          backing = data.Item?.data ? {}
-          if data.Item?
-            clock = data.Item.clock ? 0
-            lastResponseTime = data.Item.lastResponseTime
-          else
-            clock = null
-            lastResponseTime = 0
-          dirty = false
-          databaseObject =
-            isInitialized: -> return wasInitialized
-            initialize: -> wasInitialized = true
-            read: (key, markDirty) ->
-              if markDirty
-                dirty = true
-              return backing[key]
-            write: (key, value) ->
-              backing[key] = value
-              dirty = true
-              return
-            finalize: (finalizeCallback) ->
-              unless dirty
-                return setTimeout (->finalizeCallback()), 1
-
-              params =
-                TableName : tableName
-                Item:
-                  userId: DBKEY
-                  data: backing
-
-              if true
-                if clock?
-                  # item existed, conditionally replace it
-                  if clock > 0
-                    params.ConditionExpression = "clock = :expectedClock"
-                    params.ExpressionAttributeValues =
-                      ":expectedClock": clock
-                  params.Item.clock = clock + 1
-                else
-                  # item didn't exist, conditionally create it
-                  params.ConditionExpression = "attribute_not_exists(userId)"
-                  params.Item.clock = 0
-
-              dispatchSave = ->
-                #console.log "sending #{JSON.stringify(params)} to dynamo"
-                params.Item.lastResponseTime = (new Date()).getTime()
-
-                if ttlConfiguration?.AttributeName? and ttlConfiguration?.secondsToLive?
-                  params.Item[ttlConfiguration.AttributeName] = Math.round(params.Item.lastResponseTime/1000 + ttlConfiguration.secondsToLive)
-
-                dynamoDocClient.put params, (err, data) ->
-                  if err?.code == 'ConditionalCheckFailedException'
-                    console.log "DBCONDITION: #{err}"
-                    databaseObject.repeatHandler = true
-                    err = null
-                  else if err?
-                    console.error "DBWRITEFAIL: #{err}"
-
-                  finalizeCallback err, params
-
-              space = (new Date()).getTime() - lastResponseTime
-              requiredSpacing = databaseObject.responseMinimumDelay ? 500
-              if space >= requiredSpacing
-                dispatchSave()
-              else
-                wait = requiredSpacing - space
-                console.log "DELAYINGRESPONSE Spacing out #{wait}, #{(new Date()).getTime()} #{lastResponseTime}"
-                setTimeout dispatchSave, wait
-
-          fetchCallback(null, databaseObject)
-
-    else
-      mock = {}
```

Comment:
what happened to the mock table? is it no longer used?

Reply:
No need for it anymore. This was from the before before time when it was used for debugging.